From 768908db7f66195c253c84250853b3c95202a358 Mon Sep 17 00:00:00 2001
From: Nanda Krishnan <quic_nandkris@quicinc.com>
Date: Thu, 13 Oct 2022 11:54:20 +0530
Subject: [PATCH] [linux] Avoid invalidate if recycled_for_ds flag is set

Avoiding invalidation in wifi if recycled_for_ds flag is set

Change-Id: I1841d821597a27833437c203c0adc2dc4ba0001a
Signed-off-by: Nanda Krishnan <quic_nandkris@quicinc.com>
---
 include/linux/skbuff.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/include/linux/skbuff.h b/include/linux/skbuff.h
index eb4ca84a3df2..67fe0911d00d 100644
--- a/include/linux/skbuff.h
+++ b/include/linux/skbuff.h
@@ -993,6 +993,8 @@ struct sk_buff {
 	__u8			is_from_recycler:1;
 	/* Flag for fast recycle in fast xmit path */
 	__u8			fast_recycled:1;
+	/* Flag for recycle in PPE DS */
+	__u8			recycled_for_ds:1;
 	/* 1 or 3 bit hole */
 
 #ifdef CONFIG_NET_SCHED
-- 
2.34.1

